### PR TITLE
Add ability to modify uploaded videos

### PR DESCRIPTION
**Pull request by ekimekim**
Fixes https://github.com/ekimekim/wubloader/issues/170

### CODE REVIEW

File: thrimbletrimmer/edit.html [hunk 1]
```diff
@@ -196,6 +196,7 @@ <h2>Keyboard Shortcuts</h2>
 			<div id="submission-toolbar">
 				<button id="submit-button">Submit</button>
 				<button id="save-button">Save Draft</button>
+				<button id="submit-changes-button">Submit Changes</button>
```
Comment (ElementalAlchemist) [inline thrimbletrimmer/edit.html:199]: Is this meant to start hidden if the video isn't done yet?

Comment (ekimekim) [reply thrimbletrimmer/edit.html:199]: ah, yes it is. So I'd want `<button id="submit-changes-button" class="hidden">` ?

Comment (ElementalAlchemist) [reply thrimbletrimmer/edit.html:199]: Yup, that'll start it hidden (and is exactly how I'd make it hidden on page load).

**Comment by chrusher** [comment]
Late to the party but looks good to me.

**Pull request merged**